include/marl/scheduler.h

```diff
@@ -73,24 +99,36 @@ class Scheduler {
   // enqueue() queues the task for asynchronous execution.
   void enqueue(Task&& task);
 
+  // config() returns the Config used to build the schededuler.
```

Comment:
Suggest: returns the Config *that* was used to build the scheduler.

Otherwise, it sounds like it returns a Config to build a scheduler.

Reply:
Done.